.mergify.yml

```diff
@@ -13,3 +13,18 @@ pull_request_rules:
     actions:
       queue:
         name: default
+
+  - name: Automatic merge on approval (Picasso Runtime)
+    conditions:
+      - or:
+        - "#approved-reviews-by>=3"     
```

Comment:
I think we need to make this review/approval by technical committee. Merging things to Picasso runtime should not be done by just approval of 3 members.

Reply:
imo it should not be done by 2 members as it was before as it leads to issues. I did better then it was to avoid confusion in future. please raise another ticket to make it tuned more. 

my goal was to ensure that any change into picasso runtime is looked better then it was to avoid issues, not to make it perfect.

closing for now until other comments or concerns are raised. aligning MF with TC is not part of this PR. I just did thing s better, not best.